distrib/initscripts/meson.build

```diff
@@ -0,0 +1,168 @@
+if get_option('enable-redhat-sysv')
```

Comment:
How much of an effort would it be to align the initscripts parameter syntax with v3 / main? I.e. `--with-init-style=redhat-sysv`

Since we're redoing the build system anyways, it would be helpful for maintainability in the long run to consolidate behavior between the two branches where it makes sense.

Reply:
Not too difficult but then we'd end up with one init flag syntax for autotools and a different one for meson. This could cause confusion for the user during the transition. I suggest we keep the option naming syntax for autotools and meson the same until autotools is in the rear-view mirror. That day cannot come a day too soon ;)

Reply:
Fair enough. We can do another breaking change in the future when retiring autotools.